test/functional/aws_rds_resources_exist.go

```diff
@@ -51,6 +51,8 @@ func AWSRDSResourcesExistTest(t common.TestingTB, ctx *common.TestingContext) {
 }
 
 // return expected resource variables
-func verifyRDSInstanceConfig(instance rds.DBInstance) bool {
-	return *instance.MultiAZ && *instance.DeletionProtection && *instance.StorageEncrypted && *instance.AutoMinorVersionUpgrade == false && *instance.EngineVersion == "13.4"
+func verifyRDSInstanceConfig(instance rds.DBInstance, isSTS bool) bool {
+	return rdsTagsContains(instance.TagList, awsManagedTagKey, awsManagedTagValue) &&
+		(!isSTS || rdsTagsContains(instance.TagList, awsClusterTypeKey, awsClusterTypeRosaValue)) &&
+		*instance.MultiAZ && *instance.DeletionProtection && *instance.StorageEncrypted && !*instance.AutoMinorVersionUpgrade && *instance.EngineVersion == "13.4"
```

Comment:
A bit difficult to read, maybe it can be broken down more?

Reply:
Agreed, I guess I could split it up into if/else's if that helps albeit less efficient, or split the last line into multiple lines?

Reply:
I find confusing the following condition `(!isSTS || rdsTagsContains(instance.TagList, awsClusterTypeKey, awsClusterTypeRosaValue))`. It may be better with nested if, or maybe adding a comment to explain how it works. Either one is fine with me

Reply:
Sure that makes sense, I've added a comment there to clarify